src/views/tutorial-view/index.tsx

```diff
@@ -123,9 +123,7 @@ export default function TutorialView({
           />
           {isInteractive ? <InteractiveLabToggle /> : null}
         </header>
-        <Content
-          content={<MDXRemote {...content} components={MDX_COMPONENTS} />}
-        />
+        <MDXRemote {...content} components={MDX_COMPONENTS} />
```

Comment:
I believe we no longer want to use our `@hashicorp/react-content` component. For now, I think it's replaced by `src/components/dev-dot-content` (which I think we want to convert to individual MDX components).

Reply:
I don't see `DevDotContent` going away totally since some of it's global handling is meant to be so for consistency. The following block for example is easier to keep in a global setting IMO:

```css
.h1,
.h2,
.h3,
.h4,
.h5,
.h6 {
  & + ul,
  & + ol {
    margin-top: 12px;
  }

  & + p {
    margin-top: 0;
  }

  /*
  TODO: these values are a special request from design for the purposes of the
  inital preview. They make <code> elements within heading elements a little
  less jarring.
  */
  & code {
    font-size: 0.6em;
    font-weight: 500;
  }
}
```

A side note for posterity, we are using quite a few custom components already here: https://github.com/hashicorp/dev-portal/blob/main/src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.jsx

- `Tabs` and `Tab`
- `DocsAnchor` (uses `InlineLink` under the hood)
- `Heading` for all h1-h6 elements
- `Text` for p elements

Reply:
☝️ Great point, sibling selectors are hard to move out of the `:global` context.

The `code` though is to me an example of something we should move into the `Heading` component. So I think we should still try to convert `DevDotContent` to individual components where possible, so that styles are well-encapsulated and don't risk leaking into custom components (this is something we've already had a struggle contending with when styling MDX content in docs sites, and markdown content on `.com` as well).

Reply:
Right, not in disagreement about using custom components for MDX content. We've been on the same page there all along. 😊

Reply:
What would make the most sense to me is if we actually call `DevDotContent` (or whatever the name is) here, instead of at the sidebar layout level. Because we want these mdx-type content styles to be scoped to this component and not _everything_ in the layout. That would take additional refactoring though. So I'm fine with not doing it now, but any thoughts on this change? @ashleemboyer

Reply:
With this PR in mind, I think we're in a decent spot on this front, right? Plan being:

- This PR sets up custom MDX `img` → `Image` for the tutorials view (and also `ImageConfig`)
- Follow-up PR will use  `img` → `Image` on the docs view too (we'll need to reconcile some styles, this will involve moving any generic `img` styles in `DevDotContent` into the `Image` component)
- Future work will involve converting all non-sibling selector stuff in `DevDotContent` to MDX custom components, similar to the existing `Heading` and `Text`

 I've set up [an Asana task](https://app.asana.com/0/1201987349274776/1202042900781425/f) to discuss that "Future work" part. I've kicked it off with my opinions and perspective on what our acceptance criteria should be, based on past woes dealing with styling markdown and MDX across learn, docs, and dot-com via our `@hashicorp/react-content` component.

Reply:
I added a comment to the new discussion task that links to our existing task for MDX components: https://app.asana.com/0/1201987349274776/1201653123138729/f